recipe/meta.yaml

```
@@ -7,49 +7,61 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/confluentinc/confluent-kafka-python/archive/refs/tags/v{{ version }}.tar.gz
   sha256: ee099702bd5fccd3ce4916658fed4c7ef28cb22e111defb843d27633100ff065
   patches:
     - 0001-Fix-setup-for-windows.patch  # [win]
 
 build:
   number: 0
+  skip: True # [py<38 or s390x]
   script:
     - export CFLAGS="${CFLAGS} -I${PREFIX}/include"  # [unix]
     - export LDFLAGS="${LDFLAGS} -I${PREFIX}/lib"    # [unix]
     - set CFLAGS="%CFLAGS% /I%LIBRARY_INC%"          # [win]
     - set LDFLAGS="%LDFLAGS% /I%LIBRARY_LIB%"        # [win]
-    - {{ PYTHON }} -m pip install . -vv
+    - {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler('c') }}
+    - patch       # [osx and x86_64]
+    - m2-patch    # [win64]
   host:
     - python
     - setuptools
     - pip
-    - librdkafka >={{ version }}
+    - wheel
+    - librdkafka
   run:
     - python
-    - librdkafka >={{ version }}
+    - librdkafka
 
 test:
+  source_files:
+    - tests
+  requires:
+    - pip
+    - pytest
```

```suggestion
    - pytest
    - pytest-timeout
```
https://github.com/confluentinc/confluent-kafka-python/blob/v2.2.0/setup.py#L20